modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -863,7 +863,8 @@ class Session implements ISession {
     }
 
     boolean enableModuleBinaries() {
-        config.navigate('nextflow.enable.moduleBinaries', false) as boolean
+        final moduleBinaries = config.navigate('nextflow.enable.moduleBinaries', false) as boolean
+        moduleBinaries ?: NF.isModuleBinariesEnabled()
     }
```

Comment:
Since you are applying the config setting in CmdRun, I think this method can be deleted and just use `NF.isModuleBinariesEnabled()` across the board

Reply:
@jorgee following our conversation I think it should be safe to remove this method and use `NF.isModuleBinariesEnabled()` everywhere else. The same is done for e.g. strict mode